examples/monaco-graphql-webpack/src/index.ts

```diff
@@ -320,7 +321,8 @@ export function renderGithubLoginButton() {
       { provider: 'github', scope: ['user'] },
       async (err: Error, data: { token: string }) => {
         if (err) {
-          console.error('Error Authenticating with GitHub: ' + err);
+          // eslint-disable-next-line no-console
+          console.error('Error authenticating with GitHub:', err);
```

Comment:
here (and for other below cases) concatenation of `error` can produce unexpected `[object Object]` value